Assets/AnkrSDK/Examples/Scripts/UI/ConnectionController.cs

```diff
@@ -1,7 +1,7 @@
+using AnkrSDK.Plugins.WalletConnectSharp.Core;
+using AnkrSDK.Plugins.WalletConnectSharp.Unity;
+using AnkrSDK.Plugins.WalletConnectSharp.Unity.Events;
```

Comment:
I don't think that the Plugins folder should introduce a namespace.
<img width="563" alt="image" src="https://user-images.githubusercontent.com/9112595/221761438-026579b0-b605-405a-aa5a-576b22912860.png">
Uncheck this mark in folder properties, it will result in a much cleaner namespace.

Reply:
And fix namespace in folder after that